site/content/how-to/traffic-management/routing-traffic-to-your-app.md

```diff
@@ -0,0 +1,371 @@
+---
+title: "Routing Traffic to Your Application"
+description: "Learn how to route external traffic to your Kubernetes applications using NGINX Gateway Fabric."
+weight: 100
+toc: true
+docs: "DOCS-000"
+---
+
+{{<custom-styles>}}
+
+## Overview
+
+You can route traffic to your Kubernetes applications using the Gateway API and NGINX Gateway Fabric. Whether you're managing a web application or a REST backend API, you can use NGINX Gateway Fabric to expose your application outside the cluster.
+
+## Prerequisites
+
+- [Install]({{< relref "installation/" >}}) NGINX Gateway Fabric.
+- [Expose NGINX Gateway Fabric]({{< relref "installation/expose-nginx-gateway-fabric.md" >}}) and save the public IP address and port of NGINX Gateway Fabric into shell variables:
+
+   ```text
+   GW_IP=XXX.YYY.ZZZ.III
+   GW_PORT=<port number>
+   ```
+
+## Example application
+
+The application we are going to use in this guide is a simple **coffee** application comprised of one service and two pods:
+
+![coffee app](/img/route-all-traffic-app.png)
+
+Using this architecture, the **coffee** application is not accessible outside the cluster. We want to expose this application on the hostname "cafe.example.com" so that clients outside the cluster can access it.
```

Comment:
```suggestion
Using this architecture, the **coffee** application is not accessible outside the cluster. We want to expose this application on the hostname _cafe.example.com_ so that clients outside the cluster can access it.
```